apps/etterlatte-trygdetid/src/main/kotlin/trygdetid/TrygdetidRepository.kt

```diff
@@ -259,15 +259,17 @@ class TrygdetidRepository(private val dataSource: DataSource) {
         }
 
     private fun hentGrunnlagOpplysninger(trygdetidId: UUID): List<Opplysningsgrunnlag> =
```

Comment:
Burde denne bruke en inTransaction som i vedtak? Også har vi en `inTransaction` metode for de som bruker kotliquery?

Reply:
Ja, egentlig tenker jeg det er greit å få på plass, men det er en del flere endringer for å passe på at alle metodene i repositoryet respekterer transaksjonene de er innenfor.

Reply:
Og det ble fjernet i #1395, så vil ikke legge det tilbake uten videre 😅